database/migrations/2020_08_26_123347_create_h5p_elasticsearch_fields_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateH5pElasticsearchFieldsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('h5p_elasticsearch_fields', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('library_id')->unsigned();
```

Comment:
Isn't this related to the h5p_library table?
If yes, add relation.

Reply:
Added relation to h5p_library table.